lib/headlines/security_headers/x_frame_options.rb

```diff
@@ -15,7 +15,7 @@ def score_by_value
       end
 
       def score_by_allow_from
-        whitelisted_domain? ? 1 : 0
+        whitelisted_domain? ? 1 : -1
```

Comment:
@ArthurZaharov I think zero score was set by @CalebFenton after doing some tests internally. We may have forgotten to change the scoring sheet. 

@CalebFenton can you confirm what the actual score should be? I think our srcclr.com site doesn't set this header for some particular reason and based on that we had bumped the scoring a bit.